### DISCUSSION TITLE
Permission denied

### DISCUSSION THREAD

**Discussion by wardahjassiem** [nextflow-io/nextflow#2089] [Q&A]
When I try to run my pipeline on a cluster using slurm, I get the following command error: 

Can't locate Config_heavy.pl:   /usr/people/wjassiem/project/work/conda/chipseq-nf-c8e321ef0a6847733511c80b8ad8b081/lib/5.26.2/x86_64-linux-thread-multi/Config_heavy.pl: Permission denied at /usr/people/wjassiem/project/work/conda/chipseq-nf-c8e321ef0a6847733511c80b8ad8b081/lib/5.26.2/x86_64-linux-thread-multi/Config.pm line 105.

I'm using conda with nextflow, I'm not sure why i'm getting permission denied on the conda folder created by nextflow. Are there user configurations that should be done for my user name in the cluster?

**Comment by fmadani** [comment]
I faced similar issue. I am running this process which is part of my code:


process Presice_2 {
	echo true

	input:
		path R_file from Presice_ch_R
		path PCs_File from Presice_ch_PCs
		path cov_File from Presice_ch_cov
		
		path base_file from Presice_ch_base_file
		path target_file from Presice_ch_target_file
		path phenotype_file from Presice_ch_phenotype_file
		path covariate_file from Presice_ch_covariate_file
		
		
		
	"""
	Rscript $R_file  $PCs_File $cov_File  covariate_file  $params.Presice_columns_names
	
	Rscript $baseDir/PRSice_linux/PRSice.R \
		--prsice $baseDir/PRSice_linux \           <------------- This line causes the error
		--base $baseDir/$base_file \
		--target $baseDir/$target_file \
		--binary-target F \
		--pheno $baseDir/$phenotype_file \
		--cov $baseDir/$covariate_file \
		--base-maf MAF:$params.MAF \
		--base-info INFO:$params.INFO \
		--stat $params.column_name_effect_size \ 
		--or \
		--out $params.out_filename_prefix
	"""	
}

and I get this error

Command error:
  sh: 1: /mnt/c/users/fmadani/Documents/healthlligence/Precision_Medicine/Polygenic_risk_score/nextflow/PRS_Calculation/PRSice/PRSice_linux/: Permission denied
  Error:
  Execution halted
